### PR TITLE
Windriver

### PR DESCRIPTION
**Pull request by Danmcaires**

### CODE REVIEW

File: src/api_request.py [hunk 1]
```diff
@@ -174,7 +174,7 @@ def get_api_completion(self):
 
         # Create prompt
         prompt = ChatPromptTemplate.from_messages([
-        ("system", f"You are an API generator, based on the user question you will suggest the best API endpoint to retrieve the information from a StarlingX cluster.\n\nYou will look in the context for the available APIs in a StarlingX cluster.\n\nMake sure the provided endpoint is present on the provided context and check the action of the APIs to provide the ideal url for the user question.\n\nAlso make sure to only provide the API endpoint following the format: {format_response}. Guarantee that the format is followed."),
+        ("system", f"You are an API generator, based on the user question you will suggest the best API endpoint to retrieve the information from a Wind River cluster.\n\nYou will look in the context for the available APIs in a Wind River cluster.\n\nMake sure the provided endpoint is present on the provided context and check the action of the APIs to provide the ideal url for the user question.\n\nAlso make sure to only provide the API endpoint following the format: {format_response}. Guarantee that the format is followed."),
```
Comment (borges2157) [inline src/api_request.py:177]: WRCP instead of Wind River?

**Review by borges2157 — CHANGES_REQUESTED** [review]
In general, shouldn't the prompts receive WRCP instead of Wind River?

File: src/api_request.py (hunk 1 above)
Comment (Danmcaires) [reply src/api_request.py:177]: I don't like to use acronyms here, I can't use the full name. It is "Wind River Cloud Platform" right?

Comment (borges2157) [reply src/api_request.py:177]: idk. Let's keep it this way. 
Down the road we see if it turns into an issue.

**Review by borges2157 — APPROVED** [review]

**Pull request merged**